packages/yarnpkg-fslib/tests/ZipFS.test.ts

```diff
@@ -143,6 +143,25 @@ describe(`ZipFS`, () => {
     zipFs2.discardAndClose();
   });
 
+  it(`defaults the readSync read length to the buffer size`, async () => {
+    const p = `/dir/file` as PortablePath;
+    const libzip = getLibzipSync();
+    const tmpfile = ppath.resolve(xfs.mktempSync(), `test.zip` as Filename);
+    const zipFs = new ZipFS(tmpfile, {libzip, create: true});
```

Comment:
```suggestion
    const zipFs = new ZipFS(null, {libzip});
```
You can use a in-memory zip file to avoid unnecessary IO

Reply:
Thanks for that :-) Makes sense :+1: